tests/test_validation.py

```diff
@@ -31,6 +32,7 @@ class DisallowExtraFieldsSchema(Schema, DisallowExtraFieldsMixin):
     b = fields.String(load_from="c")
 
 
+class ActuallyRequireOnDumpMixinSchema(Schema, ActuallyRequireOnDumpMixin):
 @skip_if_marshmallow_not_v2
```

Comment:
Still annotated with `@skip_if_marshmallow_not_v2`. Should remove. And remove the annotation definition as well.